src/nft/components/collection/CollectionStats.tsx

```diff
@@ -234,26 +233,26 @@ const CollectionDescription = ({ description }: { description: string }) => {
 const StatsItem = ({ children, label, isMobile }: { children: ReactNode; label: string; isMobile: boolean }) => {
   return (
     <Box display="flex" flexDirection={isMobile ? 'row' : 'column'} alignItems="baseline" gap="2" height="min">
+      <span className={styles.statsValue}>{children}</span>
       <Box as="span" className={styles.statsLabel}>
         {`${label}${isMobile ? ': ' : ''}`}
```

Comment:
Mobile should be on a new line instead of having a color https://www.figma.com/file/bCN7G6oArm7pPts03jvJ5x/Unification?node-id=13682%3A86350